### ISSUE TITLE
add parallelism to OpenSSL documentation build on Windows [post 1.1.x]

### ISSUE THREAD

**Issue by sav-ix** [openssl/openssl#3401]
Hello, everyone,

For OpenSSL builds on Windows, commands `make -j<n>` (for mingw-w64) or `jom -j<n>` (for MSVC) could be used to add parallel execution for binaries build. But commands `make -j<n> install`, `make -j<n> install_docs`, etc. has no effect on OpenSSL documentation build. This make documentation build last even longer than binaries build.

Implementing parallelism for docs build could greatly speedup CI builds on Windows through utilizing all CPU cores. Although it 
> [would require a too big change in the configuration and makefile builder ... such a change for a future OpenSSL, probably 1.2.0](https://github.com/openssl/openssl/issues/1105#issuecomment-220766761)

CC:@levitte

Alexander

**Comment by levitte** [comment]
Oh my, this has been lying around for quite a bit.

I have a branch hanging around on my laptop that makes building the docs a build time thing rather than an install time one.  It's mainly a `build.info` hack, which means that the doc file build is done the same way as all other stuff, and can therefore make use of `make` parallelism.  Sounds like an idea, yeah?

**Comment by richsalz** [comment]
I believe it is more important that we (and yeah, I suppose I mainly mean the full-time fellows) to be focused on reviewing and merging existing PR's, and not looking for little baubles to polish.

**Comment by vladak** [comment]
On my laptop with Intel i7-8665U with SSD, the `install_docs` target for the current master branch takes one minute and half if the machine is otherwise idle; on regular build machines this takes much longer. Watching the graph of the CPU core utilization being flat makes me think all these `pod2man` and `mkpod2html` processes ought to run in parallel. Running full build repeatedly makes me go for a coffee more than I wanted so this is actually not a nice-to-have thing but something that would have a real world impact :-)

**Comment by nhorman** [comment]
Marking as inactive to be closed at the end of 3.4 dev barring further input

**Comment by t8m** [comment]
Closing as the issue was labeled as inactive for extended period of time and there was no additional input.
Please add a comment if you have any further information to support reopening the issue.
